modules/api/src/main/java/com/intuit/wasabi/api/ExperimentsResource.java

```diff
@@ -52,9 +66,7 @@
 import static java.lang.Boolean.FALSE;
 import static java.util.TimeZone.getTimeZone;
 import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
-import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static javax.ws.rs.core.MediaType.*;
```

Comment:
It's not a strange setting, it's just not correctly set... :-/ I want explicit for everything.